scripts/cm_test_all_sandia

```diff
@@ -1173,7 +1178,7 @@ setup_env() {
     NEW_TPL_LIST=$(echo ${NEW_TPL_LIST} | sed 's/.\w*$//')
     #echo "TESTING NEW_TPL_LIST=$NEW_TPL_LIST"
 
-    KOKKOSKERNELS_ENABLE_TPL_CMD="--with-tpls=${NEW_TPL_LIST}"
+    KOKKOSKERNELS_ENABLE_TPL_CMD="--with-tpls=${KOKKOSKERNELS_ENABLE_TPLS},${NEW_TPL_LIST}"
```

Comment:
I am not sure if Nathan intends to have users set tpls when doing the spotcheck or if only pre-decided tpls are actually enabled?
I would defer to him on that change to avoid breaking nightly tests, etc...